region/brinstar/red/Red Tower.json

```diff
@@ -470,6 +472,28 @@
       },
       "devNote": "Assume no broken runway tiles since there is never a reason to break them."
     },
+    {
+      "link": [1, 1],
+      "name": "Leave with  Moondance",
+      "notable": false,
+      "requires": [
+        "canMoondance",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 4
+        }}
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 1
+        }
+      },
+      "note": [
+        "Bring the Beetom door and begin Moondancing.",
+        "Stop after exactly 175 moonfalls, so as not to fall through the floor."
```

Comment:
Might it be possible to leave with extended Moondance by doing the moonfalls in the open door frame, where the floor is more than one tile thick?